src/include/sof/math/base2log.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_TWO_LOGARITHM_H__
+#define __SOF_MATH_BASE_TWO_LOGARITHM_H__
+
+/* Include Files */
+#include <stdint.h>
+
+/* Defines Constant*/
+#define BASE2LOG_WRAP_SCHAR_BITS 0xFF
+#define BASE2LOG_UPPERBYTES 0xFFFFFF
+#define BASE2LOG_WORDLENGTH 0x1F
```

Comment:
Move these three into beginning of base2log.c. No need for them in apps, they look purely internal to your algorithm.